spinalcordtoolbox/process_segmentation/script.py

```diff
@@ -0,0 +1,982 @@
+from sct_straighten_spinalcord import smooth_centerline
```

Comment:
You can put that in `__init__.py` except functions which are just helpers of the command-line entry point, which would go in a `script` file (which we'll rename later because it's not the best name).